COMPILING.md

```diff
@@ -380,7 +380,85 @@ For the packaged builds of CBMC on our release page we currently build CBMC
 with the MiniSat2 SAT solver statically linked at compile time. However it is
 also possible to build CBMC using alternative SAT solvers.
 
-### Compiling with Riss
+### Compiling with CBMC Wrapper
```

Comment:
:pick: Not sure I would use the phrase "CBMC Wrapper". It could suggest a third party piece of software which wraps around cbmc, rather than our intended meaning of classes which live in CBMC and wrap around the native interfaces of the solver. Maybe "Compiling CBMC using solver native interfaces." would be better.

Reply:
Updated.

Reply:
:+1: